src/cgutils.cpp

```diff
@@ -2008,18 +2008,20 @@ static jl_cgval_t typed_store(jl_codectx_t &ctx,
             emit_unbox_store(ctx, rhs, ptr, tbaa, alignment);
         }
     }
+#if JL_LLVM_VERSION >= 130000
+    else if (isswapfield && isStrongerThanMonotonic(Order)) {
+#else
     else if (isswapfield && !isboxed) {
+#endif
```

Comment:
```suggestion
    else if (isswapfield && isStrongerThanMonotonic(Order)) {
```

LLVM 13 does not need to be supported anymore (x-ref: #49722)